### PR TITLE
Testnet 0.22

### PR DESCRIPTION
**Pull request by HashEngineering**
## Issue being fixed or feature implemented

Updating a few things for Platform and Testnet 0.22

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

File: dashpay/src/main/kotlin/org/dashj/platform/dashpay/Profiles.kt [hunk 1]
```diff
@@ -181,30 +181,14 @@ class Profiles(
...
-            return documents
-        }
+        val documentQuery = DocumentQuery.builder()
+            .whereIn("\$ownerId", userIds)
+            .where("\$updatedAt", ">", timestamp)
+            .orderBy("\$ownerId", true)
+            .orderBy("\$updatedAt", true)
+            .build()
+
+        return platform.documents.get(DOCUMENT, documentQuery)
```
Comment (HashEngineering) [inline dashpay/src/main/kotlin/org/dashj/platform/dashpay/Profiles.kt:191]: returning this code to its original form which is a single query with two range operators "in" and ">" to find all profiles that match a list of identity id's that were updated after a certain time.

File: examples/src/main/kotlin/org/dashj/platform/examples/GetNetworkStatus.kt [hunk 2]
```diff
@@ -165,7 +165,7 @@ class GetNetworkStatus {
 
             println("${badNodes.size} masternodes failed to respond to one or more requests")
             println("List of non-responsive nodes:\n $badNodes")
-            println("\nList of nodes with no errors:\n\t${goodNodes}\n")
+            println("\nList of nodes with no errors:\n\t${goodNodes.map { "\"$it\"" }}\n")
```
Comment (HashEngineering) [inline examples/src/main/kotlin/org/dashj/platform/examples/GetNetworkStatus.kt:168]: this change will allow us to take the list of good nodes and copy/paste it into `TestNet3Params` in `dashj`

File: platform-core/src/main/kotlin/org/dashj/platform/sdk/platform/Platform.kt [hunk 3]
```diff
@@ -75,8 +75,6 @@ class Platform(val params: NetworkParameters) {
             get() = getAppList()
     }
 
-    val queriesSupportMultipleRanges = false // DPP 0.22 - GroveDB does not support multiple ranges in queries
-
     init {
```
Comment (HashEngineering) [inline platform-core/src/main/kotlin/org/dashj/platform/sdk/platform/Platform.kt:78]: Now Platform 0.22 supports queries with more than one range.

**Review by Syn-McJ — APPROVED** [review]
Looks good

**Pull request merged**